### PR TITLE
PHEONIX PR'S THAT READD DEPARTMENT BUDGET CARDS

### PR DESCRIPTION
**Pull request by wonderinghost**
# Document the changes in your pull request

readds physical budget cards to departments that dont have them (sci, med, engi, and service)

# Why is this good for the game?
its been known that yogs economy is in a goofy state of overt reliance on requiring a id card with a account to buy anything, yet anything in cargo is more expensive than purchasing a rcd from a vending machine. 

having budget cards back can make it easier to get your hands on physical money. while in turn to allow heads of the department to reward bonuses or charge people for a service. this is a small step but i want to see yogs pull back this rotten board and get it replaced before it causes someone to fall in instead of it being circle jerked in the discord. 

yogs has many rotten boards its built on, however this economy board should be addressed first
(also pr bounties should probably be addressed better as well, i have no idea where to put this but where people are likely to see)

# Testing
once to see if they populate contents 

# Changelog

:cl:  
rscadd: Adds long forgotten budget cards to other head lockers
/:cl:

### CODE REVIEW

**Comment by ynot01** [comment]
Use NT IRN

**Comment by Maia-J** [comment]
> Use NT IRN

NT IRN can't pay my staff a bloody bonus now can it

**Comment by ShoziOffical** [comment]
> NT IRN can't pay my staff a bloody bonus now can it

kinda totally agree with him, you cant use NT IRN to give someone a bonus paycheck (Unless you change it)

**Comment by azzzertyy** [comment]
i disagree on this
theres a major cargo rebalance that needs to happen, not this.
the economy is in a state where vending machines make sense and are pretty good and well priced, but cargo isn't, and only cargo isn't.
adding this back doesnt really solve that problem, as all the budget cards EVER got used for is when nuke ops are declared, that means we all give our budget card to cargo.

**Comment by wonderinghost** [comment]
Sir the economy removed functionality to use cash on vending machines. If you don't have a id with a account you can never use the vendors.  That's excluding the fact department people mostly get their shit free from the vendors and there are only things you can get in those vendors (looks at the rcd). 

If it is such a problem for nukies then make war ops prices inflate for the crew.

**Comment by wonderinghost** [comment]
Also if you bring traitors into this. Nerf their briefcase of cash. It's 1tc for 1000 credits. Can't really do shit with it even if you privately order because private orders still have their access locks plus making it more expensive.

**Comment by Yarinoi** [comment]
actually 1 tc is worth 666.66666.. credits, someone please nerf the briefcase to reflect the true value of tc

**Comment by azzzertyy** [comment]
> Sir the economy removed functionality to use cash on vending machines. If you don't have a id with a account you can never use the vendors. That's excluding the fact department people mostly get their shit free from the vendors and there are only things you can get in those vendors (looks at the rcd).
> 
> If it is such a problem for nukies then make war ops prices inflate for the crew.

when do you ever not have an account?
if you dont have an id, you can get one, if you dont have an account? L youre not supposed to buy shit on the station to begin with

**Comment by Maia-J** [comment]
> i disagree on this theres a major cargo rebalance that needs to happen, not this. the economy is in a state where vending machines make sense and are pretty good and well priced, but cargo isn't, and only cargo isn't. adding this back doesnt really solve that problem, as all the budget cards EVER got used for is when nuke ops are declared, that means we all give our budget card to cargo.

Any head with aa can use their departmental budget to buy guns from cargo

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -42,6 +42,8 @@
 	new /obj/item/door_remote/civillian(src)
 	new /obj/item/circuitboard/machine/techfab/department/service(src)
 	new /obj/item/storage/photo_album/HoP(src)
+	new /obj/item/card/id/departmental_budget/srv (src)
+	new /obj/item/card/id/departmental_budget/civ (src)
```
Comment (ynot01) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm:46]: Adding a civilian budget card is... interesting for sure

**Review by ynot01 — APPROVED** [review]

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm (hunk 1 above)
Comment (wonderinghost) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm:46]: Bup pointed it out when some jobs pay under civil, not service

**Comment by wonderinghost** [comment]
resolving merge conflict

**Comment by wonderinghost** [comment]
should be fixed

**Comment by wonderinghost** [comment]
ffs

**Comment by Hislyna** [comment]
Is this technically a revert pr?

**Comment by ShoziOffical** [comment]
you know you could just draft and resolve conflict instead of closing and reopening right

**Comment by Bibby0110** [comment]
> Is this technically a revert pr?

yes

**Comment by Bibby0110** [comment]
i like how we're bringing back the objectively worst part of economy. very cool

**Comment by wonderinghost** [comment]
> you know you could just draft and resolve conflict instead of closing and reopening right

I just woke up when I did that

**Comment by wonderinghost** [comment]
>  i like how we're bringing back the objectively worst part of economy. very cool

How no one can access a accounts money otherwise. Economy had been rotten for a long ass time. Yet we haven't really touched it since 2018.

And for what. Bitching about how money is used for nukies when crew can't ever possibly buy something on their own. If it is such a problem then refer to this exact same argument above this. 

Hell this server is marketed at medium roleplay but we have a community that wants to see red text on boards that are broken and rarely replaced. You saw off a person's leg when the injury was a cat scratch

**Comment by cuackles** [comment]
why is art review required here?

**Comment by wonderinghost** [comment]
I don't know. Probably from when I was updating my branch

**Comment by ToasterBiome** [comment]
This is not something we are currently looking to bring back. Departmental cards were used for two things: bolstering already powerful cargo crew orders on warops/clockies or straight up embezzlement, and the balance of this is "cargo and sec can have them but why should sci give their entire budget". 

Giving heads departmental cards will not enable them to give random people without IDs money, because they already can - being comically overpaid.

**Comment by iloveloopers** [comment]
Yes you totally can't already buy guns from cargo for nukies and clockies with department money

**Comment by ShoziOffical** [comment]
Me with AA and NT IRN:

> This is not something we are currently looking to bring back. Departmental cards were used for two things: bolstering already powerful cargo crew orders on warops/clockies or straight up embezzlement, and the balance of this is "cargo and sec can have them but why should sci give their entire budget".
> 
> Giving heads departmental cards will not enable them to give random people without IDs money, because they already can - being comically overpaid.

**Comment by wonderinghost** [comment]
> This is not something we are currently looking to bring back. Departmental cards were used for two things: bolstering already powerful cargo crew orders on warops/clockies or straight up embezzlement, and the balance of this is "cargo and sec can have them but why should sci give their entire budget".
> 
> Giving heads departmental cards will not enable them to give random people without IDs money, because they already can - being comically overpaid.

crew dont get that money. and even then it relies on a head being on, active, and participating in whatever to even cross the mind to give engineer 1 or assistant 27 a bonus, even then their salaries are far paler than the fucking department budget. whats funny is that we have a entirely unused bounty system that doesnt tie to department bank accounts. most of the shit we have for that isnt pre-installed on all pdas or it relies on a pe. 

and for that i bring back the rotten boards we are built on. science is probably the most rotted board in this frame we rely on that its just more assistants in a department. they also dont have a big reason to spend crap unless you are wanting to mess with the experimentor. 

money is used for buying food in the vendors or barely scraping by on the luxury shittle on a crew id. not a budget. crew.

**Pull request closed without merging**